README.md

```diff
@@ -1,52 +1,54 @@
+## What is NovusCore
```

Comment:
We are moving away from the name NovusCore, we started using it when we had a smaller scope that didn't involve making a general purpose MMO engine.

Please refer to it as just "Novus" in this read me.